src/NetAppFiles/NetAppFiles/Volume/NewNetAppFilesVolume.cs

```diff
@@ -97,12 +98,24 @@ public class NewAzureRmNetAppFilesVolume : AzureNetAppFilesCmdletBase
         public string CreationToken { get; set; }
 
         [Parameter(
+            ParameterSetName = FieldsParameterSet,
+            Mandatory = true,
+            HelpMessage = "The service level of the ANF volume")]
+        [Parameter(
+            ParameterSetName = ParentObjectParameterSet,
+            ValueFromPipelineByPropertyName = true,
             Mandatory = true,
             HelpMessage = "The service level of the ANF volume")]
         [ValidateNotNullOrEmpty]
-        [PSArgumentCompleter("Standard", "Premium", "Extreme")]
+        [PSArgumentCompleter("Standard", "Premium", "Ultra")]
         public string ServiceLevel { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A hashtable array which represents the export policy")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable[] ExportPolicy { get; set; }
```

Comment:
What is the schema for this parameter?  What kind of data do you expect users to provide?